wagtailio/static/sass/components/_cookie-message.scss

```diff
@@ -0,0 +1,72 @@
+.cookie {
+    $root: &;
+    @include z-index(modal);
+
+    padding: 0 10px 10px;
+    position: fixed;
+    right: auto;
+    bottom: 0;
```

Comment:
Suggestion: Use `inset` instead of bottom, right, top,  left. https://developer.mozilla.org/en-US/docs/Web/CSS/inset